tests/jobs.py

```diff
@@ -2011,3 +2012,15 @@ def test_remove_file_clears_import_error(self):
         import_errors = session.query(models.ImportError).all()
 
         self.assertEqual(len(import_errors), 0)
+
+    def test_scheduler_zip_dag_detection(self):
```

Comment:
Can you make the test more comprehensive, ie. test list_file_by_paths entirely instead of just testing for zip support?